defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+# NB. Supported RHEL versions: RHEL 8+
+#
+# Enable or disable service after configuration
+# to allow testing and verification before use.
+# NB. See ima_evm_setup if planning to use IMA.
+fapolicyd_setup_enable_service: false
+
+# trust list for fapolicyd configuration file
+# default "rpmdb,file"
+fapolicyd_setup_trust: "{{ '' if ansible_facts.distribution_version is
+                          version('8.2', '<=') else 'rpmdb,file' }}"
```

Comment:
```suggestion
                          version('8.2', '<=') and ansible_facts.os_family == 'RedHat'
                          else 'rpmdb,file' }}"
```

Reply:
`We don't need to check this because correct distro/version is checked in System check.`

Also applies here.